src/components/Button/DropdownButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DropdownButton as BsDropdownButton } from 'react-bootstrap';
+import { BUTTON_BS_STYLES } from './constants';
+
+const DropdownButton = props => <BsDropdownButton {...props} />;
+
+DropdownButton.propTypes = Object.assign(BsDropdownButton.propTypes, {
+  bsStyle: PropTypes.oneOf(BUTTON_BS_STYLES),
+});
```

Comment:
This modifies also the BsDropdownButton.propTypes. To make it correct you either need to use
```
DropdownButton.propTypes = Object.assign({}, BsDropdownButton.propTypes, {
  bsStyle: PropTypes.oneOf(BUTTON_BS_STYLES),
});
```
or much simpler:
```
DropdownButton.propTypes = {
  ...BsDropdownButton.propTypes,
  bsStyle: PropTypes.oneOf(BUTTON_BS_STYLES),
});
```

Reply:
@jtomasek @sharvit works for me. thanks.